iso15118/secc/comm_session_handler.py

```diff
@@ -189,21 +195,47 @@ async def __init__.
         """
 
         self.udp_server = UDPServer(self._rcv_queue, self.config.iface)
+        self.status_event_list.append(self.udp_server.ready_event)
+
         self.tcp_server = TCPServer(self._rcv_queue, self.config.iface)
+        self.status_event_list.append(self.tcp_server.tls_ready_event)
 
         self.list_of_tasks = [
             self.get_from_rcv_queue(self._rcv_queue),
             self.udp_server.start(),
             self.tcp_server.start_tls(),
+            self.check_status_task(),
         ]
 
         if not self.config.enforce_tls:
             self.list_of_tasks.append(self.tcp_server.start_no_tls())
+            self.status_event_list.append(self.tcp_server.tcp_ready_event)
 
         logger.info("Communication session handler started")
 
         await wait_for_tasks(self.list_of_tasks)
 
+    def check_events(self) -> bool:
+        result: bool = True
+        for event in self.status_event_list:
+            if event.is_set() is False:
+                result = False
+                break
+        return result
+
+    async def check_ready_status(self) -> None:
+        # Wait until all flags are set
+        while self.check_events() is False:
+            await asyncio.sleep(0.1)
```

Comment:
I think 100 ms maybe too liberal for this - as servers could potentially be ready in just a few ms - modify this to be 10ms maybe?

Reply:
Sure.